gulp/isotest.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+"use strict";
+
+module.exports = (gulp, plugins, blueprint) => {
+    const rs = require("run-sequence").use(gulp);
+    const mocha = require("gulp-mocha");
+
+    blueprint.projectsWithBlock("isotest").forEach((project) => {
+        gulp.task(`isotest-${project.id}`, [`typescript-compile-${project.id}`], (done) => {
```

Comment:
please remove the typescript dependency here. it's going to end up compiling each project multiple times throughout the `build` flow. instead, add a single dependency on `typescript-compile` below when you define the `isotest` wrapper.

or even better, don't add any dependencies and just assume that we know to compile before running tests (that's a safe assumption)

case in point: at this point in the circle build we've already run `gulp build` and now you're asking to _re-run_ the most time-consuming parts of that build (`tsc`)